snap/snapcraft.yaml

```diff
@@ -19,39 +19,101 @@ slots:
 
 apps:
   fractal:
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     command: usr/bin/fractal
     common-id: org.gnome.Fractal
     desktop: usr/share/applications/org.gnome.Fractal.desktop
     plugs:
-      - audio-playback
       - home
-      - gsettings
-      - network
-      - opengl
-      - password-manager-service
+      - removable-media
 
 parts:
+  ffmpeg:
+    # WORKAROUND:
+    # Build from source because the ffmpeg package installs libraries as dependencies
+    # that conflict with the Gnome extension
+    plugin: autotools
+    source: https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz
+    source-checksum: sha256/57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
+    autotools-configure-parameters:
+      # WORKAROUND: Install to /usr instead of /usr/local because it's not in search paths
+      - --prefix=/usr
+      - --disable-debug
+      - --disable-doc
+      - --disable-static
+      - --enable-gpl
+      - --enable-shared
+      - --disable-ffplay
+      - --disable-devices
+      - --enable-gnutls
+      - --enable-libmp3lame
+      - --enable-libvorbis
+    build-packages:
+      - nasm
+      - libgnutls28-dev
+      - libmp3lame-dev
+      - libvorbis-dev
+    stage-packages:
+      - libmp3lame0
+    stage:
+      - -usr/include
+  gtksourceview:
+    source: https://gitlab.gnome.org/GNOME/gtksourceview.git
+    source-branch: 'gtksourceview-4-6'
+    source-depth: 1
+    plugin: meson
+    meson-parameters:
+     - --prefix=/usr
+     - -Ddebug=true
+     - -Doptimization=3
+    build-packages:
```

Comment:
Are you sure we need these build packages?  These are both included in the sdk snap